src/main/java/AddList.java

```diff
@@ -0,0 +1,71 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class AddList {
+    List<Mission> ls;
+    int counter = 0;
+    public AddList() {
+        ls = new ArrayList<Mission>();
+    }
+    public void printAllEvent() {
+        int counter = 1;
+        for(Mission a : ls) {
+            System.out.print(counter + ". ");
+            ls.get(counter - 1).printMission();
+            counter += 1;
+        }
+    }
+
+    public void printEvent(int index) {
+        ls.get(index).printMission();
+    }
+
+    public void addEventWithTime(String str, String type, String time, String atby) {
+        ls.add(new Mission(str, type, time, atby));
+        counter += 1;
+    }
+
+    public void addEventWithoutTime(String str, String type) {
+        ls.add(new Mission(str, type));
+        counter += 1;
+    }
+
+    public void changeEvent(int index) {
+        ls.get(index).changeState();
+    }
+
+    public void deleteMission(int index) {
+        ls.remove(index);
+    }
+
+    public void saveToFile() {
+        try {
+            FileWriter fw = new FileWriter("duke.txt");
```

Comment:
If you have time, it's suggested to consider moving this part to Storage class for better code management, which is recommended by CS2103T website.